### ISSUE TITLE
Java8 launch fails with fatal error for image built using 5.16.0

### ISSUE THREAD

**Issue by abonstu** [paketo-buildpacks/java#309]
## What happened?

* What were you attempting to do?
Launch an image built using 5.16.0

* What did you expect to happen?
I expected the Java app to start

* What was the actual behavior? Please provide log output, if possible.
Java launch fails with the following output:
```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f90f6ffe2f7, pid=1, tid=0x00007f90f8166700
#
# JRE version:  (8.0_302-b08) (build )
# Java VM: OpenJDK 64-Bit Server VM (25.302-b08 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# V  [libjvm.so+0xb2a2f7]  VirtualMemoryTracker::add_reserved_region(unsigned char*, unsigned long, NativeCallStack const&, MemoryType, bool)+0x77
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /workspace/hs_err_pid1.log
#
# If you would like to submit a bug report, please visit:
#   https://bell-sw.com/support
#

[error occurred during error reporting , id 0xb]
```

## Build Configuration

* What platform (`pack`, `kpack`, `tekton` buildpacks plugin, etc.) are you
  using? Please include a version.

pack [0.20.0+git-66a4f32.build-2668] command executed on locally on macOS 11.6 and in CI on Ubuntu 18.04:
```
pack build "$dockerhub_snapshot_image" \
  --env BP_JVM_VERSION=8 \
  --buildpack paketo-buildpacks/java \
  --builder paketobuildpacks/builder:base \
  --path $jar_path
```

* What buildpacks are you using? Please include versions.
Using 5.15.0 image is built successfully and launches successfully.
Using 5.16.0 image is built successfully but fails to start (with output shown above)

**Comment by dmikusa-pivotal** [comment]
tl;dr there is a bug in the JVM that gets triggered by a recent config change. You can work around this by disabling the config option. Details at this issue: https://github.com/paketo-buildpacks/bellsoft-liberica/issues/131.
